web-ide/web_ide.py

```diff
@@ -168,9 +169,19 @@ async def on_shutdown():
     if dirpath.exists():
         shutil.rmtree(dirpath)
 
+async def export(q: Q):
+    tmp_files = [os.path.join(tmp_dir, file) if file.endswith(('.py')) else None for file in os.listdir(tmp_dir)]
+    with ZipFile('tmp_project/app.zip','w') as zip:
+        for file in tmp_files:
+            zip.write(file)
```

Comment:
Would [this](https://stackoverflow.com/questions/1855095/how-to-create-a-zip-archive-of-a-directory) work instead?

Reply:
Zip the whole folder with no exceptions.